modules/virt-adding-virtio-drivers-vm-yaml.adoc

```diff
@@ -8,28 +8,24 @@
 [id="virt-adding-virtio-drivers-vm-yaml_{context}"]
 = Adding VirtIO drivers container disk to a virtual machine
 
-{VirtProductName} distributes VirtIO drivers for Microsoft Windows as a
-container disk, which is available from the
-link:https://access.redhat.com/containers/#/registry.access.redhat.com/container-native-virtualization/virtio-win[Red Hat Ecosystem Catalog].
-To install these drivers to a Windows virtual machine, attach the
-`container-native-virtualization/virtio-win` container disk to the virtual machine as a SATA CD drive
-in the virtual machine configuration file.
+Add a VirtIO drivers container disk to a Windows virtual machine (VM) as a SATA CD drive.
```

Comment:
This is a little bare. From the mod docs guidance:

> The procedure introduction is a short paragraph that provides an overview of the module. The introduction includes what the module will help the user do and why it will be beneficial to the user. It includes key words that relate to the module to maximize search engine optimization.
>
> The introduction typically provides context for the procedure, such as:
> 
>   * Why and where the user performs the procedure
>   * Special considerations specific to the procedure